dojo/1.11/store.d.ts

```diff
@@ -261,7 +261,19 @@ declare namespace dojo {
 			/**
 			 * This is a basic in-memory object store. It implements dojo/store/api/Store.
 			 */
-			new <T extends Object>(options?: MemoryOptions<T>): Memory<T>;
+			new <T extends Object>(options?: MemoryOptions<T>): T & Memory<T>;
+		}
+
+		interface Observable<T> extends dojo.Stateful {
```

Comment:
Observable is a bit more complicated than this expresses.  I think I got it properly typed in 020de14d6d8bebabe2340eed2a863baedc873f9f.